docs/platforms/dart/configuration/options.mdx

```diff
@@ -196,3 +196,35 @@ Transports are used to send events to Sentry. Transports can be customized to so
 Switches out the transport used to send events. How this works depends on the SDK. It can, for instance, be used to capture events for unit-testing or to send it through some more complex setup that requires proxy authentication.
 
 </ConfigKey>
+
+## Tracing Options
+
+<ConfigKey name="enable-tracing">
+
+A boolean value, if true, transactions and trace data will be generated and captured. This will set the <PlatformIdentifier name="traces-sample-rate" /> to the recommended default of 1.0 if <PlatformIdentifier name="traces-sample-rate" /> is not defined. Note that <PlatformIdentifier name="traces-sample-rate" /> and <PlatformIdentifier name="traces-sampler" /> take precedence over this option.
+
+</ConfigKey>
+
+<ConfigKey name="traces-sample-rate">
+
+A number between 0 and 1, controlling the percentage chance a given transaction will be sent to Sentry. (0 represents 0% while 1 represents 100%.) Applies equally to all transactions created in the app. Either this or <PlatformIdentifier name="traces-sampler" /> must be defined to enable tracing.
```

Comment:
only if `enableTracing` is set to `true` - then `tracesSampleRate` will be set to the recommended value of `1.0`.

I know the wording for `enableTracing` that we copy-pasted and now have on all SDK docs is a bit confusing:

> A boolean value, if true, transactions and trace data will be generated and captured. This will set the traces-sample-rate to the recommended default of 1.0 if traces-sample-rate is not defined

The default value is `0.0`. The recommended value if you enable tracing is `1.0`. We should probably re-word to something like

> A boolean value that determines whether tracing is enabled and trace data will be generated and captured. This will set the traces-sample-rate to the recommended value of 1.0 if traces-sample-rate is not defined

OTOH there is an ongoing [DACI](https://www.notion.so/sentry/DACI-Deprecate-enableTracing-in-SDKs-cae2732d252d41d3ac5cf65038ed78a3) to decide if we remove `enableTracing` altogether